sdk/sqlmanagement/Azure.ResourceManager.Sql/api/Azure.ResourceManager.Sql.netstandard2.0.cs

```diff
@@ -2205,7 +2205,7 @@ public partial class RecommendedActionCollection : Azure.ResourceManager.Core.Ar
     public partial class RecommendedActionData : Azure.ResourceManager.Models.ResourceData
     {
         public RecommendedActionData() { }
-        public System.Collections.Generic.IReadOnlyDictionary<string, object> Details { get { throw null; } }
+        public System.Collections.Generic.IReadOnlyDictionary<string, System.BinaryData> Details { get { throw null; } }
```

Comment:
It is hard to believe that this details will always be a nested dictionary - I believe Dictionary<string, object> makes more sense here